dist/stat_data.py

```diff
@@ -384,6 +388,12 @@ def __init__(self, name, desc, flags=''):
     DhandleStat('dh_sweep_tod', 'connection sweep time-of-death sets'),
     DhandleStat('dh_sweeps', 'connection sweeps'),
 
+    ##########################################
+    # History statistics
+    ##########################################
+    HistoryStat('hs_gc_pages_visited', 'history pages visited for garbage collection'),
+    HistoryStat('hs_gc_pages_removed', 'history pages removed for garbage collection'),
+
```

Comment:
How about adding one more stat to indicate `how many pages are added for eviction`? Having this information also will help to identify easily how the GC is doing.

Reply:
@kommiharibabu is this new stat you're requesting really part of the changes that are under review already?

Reply:
@sue, No. The requested stat code already exists.  Currently the code either `delete` the page or `add it to urgent eviction` queue.

Reply:
FYI my username is not @ sue. Someone else got that notification!

Reply:
@kommiharibabu I've added the new stat and pushed, please check it.